service/lexruntimeservice/internal/benchmark/get_session_test.go

```diff
@@ -0,0 +1,115 @@
+package benchmark
+
+import (
+	"bytes"
+	"context"
+	"io/ioutil"
+	"net/http"
+	"path/filepath"
+	"testing"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	smithyClient "github.com/aws/aws-sdk-go-v2/service/lexruntimeservice"
```

Comment:
Nit: smithyClient might confuse customers who don't know what smithy is (Although I don't think they will see internal/benchmark). For the latest v2 sdk clients, I think it's safe to use v2Client.

Reply:
This is a benchmarking package for developers, not really meant to be examples for customers to use.